package-lock.json

```diff
@@ -16604,7 +16604,7 @@
         "graphql": "^15.5.0",
         "graphql-request": "^3.4.0",
         "lodash": "^4.17.21",
-        "node-fetch": "2.6.7"
+        "node-fetch": "^2.6.1"
```

Comment:
can you please revert the changes made to this file?

Reply:
reverted those changes, PR should be ready to merge